appveyor.yml

```diff
@@ -11,4 +20,4 @@ install:
     - C:\msys64\usr\bin\bash -l /c/projects/pioneer/CI/appveyor/%TARGET%/prepare.sh
 
 build_script:
-    - C:\msys64\usr\bin\bash -l /c/projects/pioneer/CI/appveyor/%TARGET%/build.sh
+    - C:\msys64\usr\bin\bash -l /c/projects/pioneer/CI/appveyor/%TARGET%/build.sh
```

Comment:
No new line?

Reply:
Yeah I don't know what happened there, I'll fix it when/if I have to update the file agian